python/test/extrinsics/test_microdata_faithfulness.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+import opendp.prelude as dp
+
+from ..helpers import optional_dependency
+
+dp.enable_features("contrib", "floating-point")
+
+
+def test_microdataa_faithfulness():
```

Comment:
Slight preference for parameterizing the test. Only 2 conditions tested, so not a big win, but it does give us clearer feedback about any failure, and makes sure all tests are tried, even if one fails
```suggestion
@pytest.mark.parametrize("faithfulness,similarity", [
    (1, lambda x, y: 2*(x != y).any()),
    (2, {"A": 1/100, "B": 1/2})])
def test_microdataa_faithfulness(faithfulness, similarity):
```

See also slight preference for handling these as separate kwargs, so it might be more like:
```suggestion
@pytest.mark.parametrize("faithfulness,similarity_function,similarity_dict", [
    (1, lambda x, y: 2*(x != y).any(), None),
    (2, None, {"A": 1/100, "B": 1/2})])
def test_microdataa_faithfulness(faithfulness, similarity_function, similarity_dict):
```